docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,414 @@
+
+#### Install `semgrep`
+
+If you cannot use the Semgrep docker image, do the following steps.
+
+1. Run `pip install semgrep` inside the container or CI environment
+2. Run `semgrep ci --config auto`.
+
+Your customization options with other CI providers vary depending on working environment variables.
+
+## Refining the CI job
+
+The following sections describe methods to customize your CI job.
+
+### Passing the CI job 
+
+By default, Semgrep CI exits with exit code 1 if the scan returns any findings. This causes the job to fail.
+
+Semgrep provides a **fail open** option. This enables you to suppress findings or internal Semgrep errors that block your pipeline.  Use any of the following commands:
+
+<dl>
+	<dt><code>semgrep ci</code></dt>
+	<dd>The Semgrep CI job<strong>fails</strong> on blocking findings or on internal errors.</dd>
+	<dt><code>semgrep ci || [ $? != 1 ]</code></dt>
+	<dd>The Semgrep CI job <strong>fails</strong> on blocking findings, but <strong>passes</strong> on internal errors. </dd>
+	<dt><code>semgrep ci || true</code></dt>
+	<dd>The Semgrep CI job <strong>passes</strong> on blocking findings and on internal errors. </dd>
+</dl>
+
+Refer to [Semgrep exit codes](../cli-reference/#exit-codes) to understand various internal issues that cause Semgrep to fail.
+
+### Setting a scan schedule
+
+The following table is a summary of methods and resources to set up schedules for different CI providers.
+
+TODO
+
+### Customizing rules and rulesets
+
+#### Exploring Semgrep Registry for useful rulesets
+
+`semgrep ci` accepts a list of rules and rulesets to run on each scan. The list is delimited by a space (` `) if the variable is exported from a command or script block, such as in the BitBucket Pipeline example. The list can also be delimited by a newline if the `SEMGREP_RULES` variable is declared through YAML syntax.
+
+
+Example snippet for space-delimited list using `export SEMGREP_RULES`:
+
+Example snippet for newline-delimited list using `SEMGREP_RULES` YAML syntax:
+
+
+#### Writing your own rules
+
+Write custom rules to enforce your team's coding standards and security practices. Rules can be forked from existing community-written rules.
+
+See [Writing rules](https://semgrep.dev/docs/writing-rules/overview/) to learn how to write custom rules.
+
+TODO
+
+
+### Ignoring files
+
+By default Semgrep CI skips files and directories such as tests/, node_modules/, and vendor/. It uses the same default .semgrepignore as the CLI, which can be found in the [CLI Reference](https://semgrep.dev/docs/cli-reference/#ignoring-files). This is used by Semgrep CI when no explicit .semgrepignore file is found in the root of your project.
+
+Use a `.semgrepignore` file to exclude files and directories from a scan. The `.semgrepignore` file follows `.gitignore` syntax. You can exclude vendored code and code for tests through this feature.
```

Comment:
is this an additional .semgrepignore from the previous paragraph? Or should this paragraph be above the previous one?

Reply:
I think I should have added something like "You can make a .semgrepignore file of your own. When Semgrep detects a user-defined .semgrepignore file within the root of your repository, it stops using the default .semgrepignore file above." 

Does that help?

Reply:
yes, but where does the default .semgrepignore file live?

Reply:
(Added it in. Resolving!)